regression-test/suites/correctness_p0/test_xor_pred.groovy

```diff
@@ -0,0 +1,42 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_xor_pred") {
+    sql """ drop table if exists dbxor; """
+    sql """
+    CREATE TABLE IF NOT EXISTS dbxor (
```

Comment:
The test case is too simple. Add such cases
1. int xor with negative number.
2. decimal xor with int or decimal.
3. datetime xor with datetime.

Reply:
Because this only accepts parameters of type bool, decimal, and datetime are cast to bool for computation.

Reply:
> The test case is too simple. Add such cases
> 
> 1. int xor with negative number.
> 2. decimal xor with int or decimal.
> 3. datetime xor with datetime.


This only accepts bool types; all other types will be cast to bool for computation.